### PR TITLE
Add possibility to set up php and tools

### PR DESCRIPTION
**Pull request by luislard**
**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)

**What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Adds possibility to set up PHP with tools in the pipeline

**What is the current behavior?** (You can also link to an open issue here)
There is no possibility to add PHP and tools

**What is the new behavior (if this is a feature change)?**
It is possible to add PHP and tools

**Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

**Other information**:
This PR is a consequence of #79

### CODE REVIEW

File: .github/workflows/build-and-push-assets.yml [hunk 1]
```diff
@@ -53,6 +53,16 @@ on:
         type: string
         default: ''
         required: false
+      PHP_VERSION:
+        description: PHP version with which the WP CLI is going to be executed.
```
Comment (shvlv) [inline .github/workflows/build-and-push-assets.yml:57]: Perhaps it should be something more generic.
```suggestion
        description: PHP version with which PHP tools are going to be executed.
```

File: .github/workflows/build-and-push-assets.yml [hunk 2]
```diff
@@ -53,6 +53,16 @@ on:
...
+        default: "8.0"
+        required: false
+        type: string
+      PHP_TOOLS:
+        description: "PHP tools to be installed. See: shivammathur/setup-php GH Action"
```
Comment (tyrann0us) [inline .github/workflows/build-and-push-assets.yml:62]:
```suggestion
        description: PHP tools supported by shivammathur/setup-php to be installed.
```

File: .github/workflows/build-and-push-assets.yml [hunk 3]
```diff
@@ -53,6 +53,16 @@ on:
...
+        default: ""
```
Comment (tyrann0us) [inline .github/workflows/build-and-push-assets.yml:63]:
```suggestion
        default: ''
```

File: .github/workflows/build-and-push-assets.yml [hunk 4]
```diff
@@ -53,6 +53,16 @@ on:
         type: string
         default: ''
         required: false
+      PHP_VERSION:
+        description: PHP version with which the WP CLI is going to be executed.
+        default: "8.0"
```
Comment (tyrann0us) [inline .github/workflows/build-and-push-assets.yml:58]:
```suggestion
        default: '8.0'
```

File: .github/workflows/build-and-push-assets.yml (hunk 2 above)
Comment (luislard) [reply .github/workflows/build-and-push-assets.yml]: Aligned

File: .github/workflows/build-and-push-assets.yml (hunk 3 above)
Comment (luislard) [reply .github/workflows/build-and-push-assets.yml]: changed

File: .github/workflows/build-and-push-assets.yml (hunk 4 above)
Comment (luislard) [reply .github/workflows/build-and-push-assets.yml]: changed

**Review by Chrico — APPROVED** [review]
LGTM 👍🏻

**Review by tyrann0us — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**